packages/core/src/messagingProtocols/proteus/ProteusService/ProteusService.mocks.ts

```diff
@@ -38,11 +38,11 @@ export const buildProteusService = async (
     clientId: getUUID(),
   };
 
-  const coreCrypto = await CoreCrypto.deferredInit('store-name', 'key');
+  const cryptoClient = new CoreCryptoWrapper({} as any, {} as any, {} as any);
```

Comment:
We can now use the generic wrapper around CoreCrypto and Cryptobox, no need for a specific implementation here :)